maven/lib/dependabot/maven/update_checker/version_finder.rb

```diff
@@ -57,8 +57,10 @@ def versions
           version_details =
             repositories.map do |repository_details|
               url = repository_details.fetch("url")
-              dependency_metadata(repository_details).
-                css("versions > version").
+              xml = dependency_metadata(repository_details)
+              next [] unless xml
```

Comment:
I spent more time parsing this than I was expecting. Part of this is that I'm less used to seeing return values with `next` (and `break`), but that's 100% valid. The other part, I think, is the way it reads as a line. 👇 is easier for _me_ to parse. What do you think of it?

```suggestion
              next [] if xml.blank?
```